pkgs/build-support/docker/default.nix

```diff
@@ -133,6 +141,7 @@ rec {
         --tmpdir=$TMPDIR \
         --override-os ${os} \
         --override-arch ${arch} \
+        --override-variant ${variant} \
```

Comment:
Seems like this would produce `--override-variant \`, consuming `copy` on the following line as a variant.
I think this needs `${optionalString (variant != "") "--override-variant ${variant}"} \`, though I'd also recommend `null` instead of `""`.

Reply:
Done :heavy_check_mark: